src/Select.js

```diff
@@ -599,9 +600,7 @@ export default class Select extends Component<Props, State> {
     }
     if (this.props.isDisabled) return;
     const { isMulti, menuIsOpen } = this.props;
-    if (!this.focused) {
```

Comment:
`this.focused` didn't exist in the class so now we just call `this.focusInput()` everytime